db/seeds.rb

```diff
@@ -1,23 +1,19 @@
-if defined?(Refinery::User)
-  Refinery::User.all.each do |user|
-    if user.plugins.where(:name => 'refinerycms_search').blank?
-      user.plugins.create(:name => 'refinerycms_search')
-    end
+Refinery::User.all.each do |user|
+  if user.plugins.where(name: 'refinerycms_search').blank?
+    user.plugins.create(name: "refinerycms_search",
+                        position: (user.plugins.maximum(:position) || -1) +1)
   end
-end
+end if defined?(Refinery::User)
 
-if defined?(Refinery::Page)
-  unless Refinery::Page.where(:menu_match => "^/search.*$").any?
-    page = Refinery::Page.create(
-      :title => "Search Results",
-      :show_in_menu => false,
-      :link_url => "/search",
-      :deletable => false,
-      :menu_match => "^/search.*$"
-    )
+if defined?(Refinery::Page) and !Refinery::Page.exists?(link_url: (url = Refinery::Search.page_url))
```

Comment:
There is no usecase at this time, 

If `Refinery::Page` is missing, the search page will be not created.

It's like the `refinerycms-blog` engine.